ui/config-overrides.js

```diff
@@ -1,3 +1,13 @@
-const { useBabelRc, override, useEslintRc } = require('customize-cra');
+const {
+  useBabelRc,
+  override,
+  useEslintRc,
+  addWebpackPlugin
+} = require('customize-cra');
+const CompressionPlugin = require('compression-webpack-plugin');
 
-module.exports = override(useBabelRc(), useEslintRc());
+module.exports = override(
+  useBabelRc(),
+  useEslintRc(),
+  addWebpackPlugin(new CompressionPlugin())
```

Comment:
Add a `,` at the end so later commits won't change this line for no good reason.

Reply:
I just add the rule on .prettierc that will also enforce that rule in the entire project :)